yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -182,7 +187,7 @@ class WrestleUniverseVODIE(WrestleUniverseBaseIE):
 
     def _real_extract(self, url):
         lang, video_id = self._match_valid_url(url).group('lang', 'id')
-        metadata = self._download_metadata(url, video_id, lang, 'videoEpisodeFallbackData')
+        metadata = self._download_metadata(url, video_id, lang, ('videoEpisodeFallbackData',))
```

Comment:
revert after applying suggestion for `_download_metadata`
```suggestion
        metadata = self._download_metadata(url, video_id, lang, 'videoEpisodeFallbackData')
```

Reply:
This's still applicable if u use https://github.com/yt-dlp/yt-dlp/pull/7432/files#r1244283283